Tests/Editor/Unity.ProBuilder.Editor.Tests.asmdef

```diff
@@ -5,18 +5,30 @@
         "Unity.ProBuilder",
         "Unity.ProBuilder.Editor",
         "Unity.ProBuilder.Stl",
-        "Unity.Settings.Editor"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Settings.Editor",
+        "Unity.PerformanceTesting",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
```

Comment:
I'm not sure that these changes will compile pre-2019.1.

Reply:
Do I need to push that file? My code takes account of whether or not we have those packages anyway

Reply:
Yes, you will need to update the asmdef with the optional dependencies.  Looking at yamato it seems like the point of failure is the NUnit dll not being found - https://yamato-artifactviewer.prd.cds.internal.unity3d.com/0e7f19c6-06b4-4b68-9dad-38fc9a7068ee%2Flogs%2Fupm-ci~%2Ftest-results%2Fisolation-2265397-com.unity.probuilder/TestReportV1.html

You might try leaving your changes but including the
```
"optionalUnityReferences": [ "TestAssemblies" ]
```
key from the original.